BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -215,8 +216,25 @@ class CbrainTask::<%= name %> < <%= descriptor['cbrain:inherits-from-class'] ||
       %>
 %     end
     }
+%   end
 
+%   unless seps.empty?
+    # Input command-line separators (with flags) used with keys in command-line
+    # substitution.
+    seps = {
+%     key_width = max_width.(seps, 'command-line-key')  + "''".length
+%     seps.each do |sep|
+      <%=
+        "%-#{key_width}s => %s" % [
```

Comment:
Theoretically they could, I suppose. I think the same thing could happen with the code just above it as well (since that is what I based that part on). In some sense I guess it requires that the keys/values used in the Boutiques json schemas for these fields do not have dangerous characters like that.